deployment/frontend/src/components/home/HighlightsCarousel.tsx

```diff
@@ -1,123 +1,128 @@
-import { Navigation } from "swiper/modules";
-import { Swiper, SwiperSlide } from "swiper/react";
-import "swiper/css";
-import "swiper/css/navigation";
-import "swiper/css/pagination";
-import CarouselNavButton from "../_shared/CarouselNavButton";
-import HighlightCard from "../HighlightCard";
-import { AutoCarousel } from "../_shared/AutoCarousel";
+import { Navigation } from 'swiper/modules'
+import { Swiper, SwiperSlide } from 'swiper/react'
+import 'swiper/css'
+import 'swiper/css/navigation'
+import 'swiper/css/pagination'
+import CarouselNavButton from '../_shared/CarouselNavButton'
+import HighlightCard from '../HighlightCard'
+import { AutoCarousel } from '../_shared/AutoCarousel'
+import { WriDataset } from '@/schema/ckan.schema'
 
 export interface TopicProps {
-  title: string;
-  numOfDatasets: number;
-  img: string;
+    title: string
+    numOfDatasets: number
+    img: string
 }
 
 const highlights = [
```

Comment:
are you still hard coding these values?

Reply:
This component just uses another component that I had to change, it's part of a page that's not wired up